apisix/plugins/authz-keycloak.lua

```diff
@@ -83,6 +84,9 @@ local schema = {
                 {
                     properties = {
                         lazy_load_paths = {enum = {false}},
+                    },
+                    anyOf = {
+                        {required = {"permissions"}}
```

Comment:
We don't need to require it as we already provide a default value

Reply:
got